src/modules/auth/components/auth-import.jsx

```diff
@@ -27,6 +30,10 @@ export default class AuthImport extends Component {
     window.addEventListener('error', this.handleRecoverError);
   }
 
+  componentWillUnmount() {
+    window.removeEventListener('error', this.handleRecoverError);
```

Comment:
There was an uncaught error coming back from Keythereum if the file or pass was bad I believe, so this catches that and displays an appropriate error.